build.sbt

```diff
@@ -387,11 +387,6 @@ lazy val `cloud-nio-impl-drs` = (project in cloudNio / "cloud-nio-impl-drs")
   .dependsOn(common)
   .dependsOn(common % "test->test")
 
-lazy val perf = project
-  .withExecutableSettings("perf", dependencies = perfDependencies, pushDocker = false)
-  .dependsOn(common)
-  .dependsOn(common % "test->test")
```

Comment:
This project has been unused for a few years but was still diligently building its Docker image every time I did a global `sbt docker`.

Reply:
best way to optimize build times